### PR TITLE
Add whatsnew page for Firefox 97 [fix #11172]

### LINKED ISSUE #11172: WNP 97
## Description

Create two what's new pages for Firefox 97, releasing on *8 February, 2022*.

FR and DE will show the "future of the internet" video.
- [Copy](https://docs.google.com/document/d/14lqnSGPOenKX_n22dc_-UKUHJ6Gojo0sLjei8rbDMEQ/)
- [Design](https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/Firefox-WNP-EU-(Copy-June-2021)?node-id=1308%3A292)

EN will promote Pocket.
- [Copy](https://docs.google.com/document/d/1PkOfO0cHRFHNwSSa4e_qh9w9hhAnzC-_wWf2zdtp71o/)
- [Design](https://www.figma.com/file/CZvzHgU9NQMQUm34lomIl5/Firefox-WNP?node-id=1598%3A39533)

### PR DESCRIPTION
**Pull request by craigcook**
## Description
WNP97, promoting Pocket for English and "The Big Picture" video for French and German (they also have promos for VPN and Pocket). Release date is **8 February, 2022** (next Tuesday) so we need to get this live before then.

## Issue / Bugzilla link
#11172 

## Testing
http://localhost:8000/en-US/firefox/97.0/whatsnew/
Pocket button should go through the FxA flow (asking you to create an account if you're signed out, or recognizing you're signed into Firefox and prompting you to sign into Pocket).

http://localhost:8000/de/firefox/97.0/whatsnew/
http://localhost:8000/fr/firefox/97.0/whatsnew/
Main CTA button should open a modal and play the video with captions turned on, and captions should be the correct language.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <div class="mzp-l-content mzp-t-content-nospace">
+      <div class="video-backdrop mzp-t-dark">
+        <div class="video-backdrop-inner">
+          <h1 class="wnp-main-title">Blicke mit uns auf das große Ganze</h1>
+          <p class="wnp-main-tagline">
+            Zusammen mit einer globalen Gemeinschaft von Machern und Denkern behalten wir das große Ganze im Auge. Um dir zu helfen, dich im Web zurechtzufinden und die Zukunft des Internets gemeinsam zu gestalten.
+          </p>
+
+          <p class="wnp-main-cta">
+            <a href="https://www.youtube.com/watch?v=_obdx3VUzuQ&cc_lang_pref=de&cc_load_policy=1" class="mzp-c-button mzp-t-product mzp-t-lg js-video-play">▶ Video abspielen</a>
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html:35]: question (non-blocking): is it worth adding some GTM data attributes on this link to open the modal? (we already have the video-interaction data from the JS, so maybe not necessary here)

**Review by maureenlholland — COMMENTED** [review]
@craigcook I need some info on the UTM and GTM expectations for these pages before I approve.

Otherwise, everything looks good and functions as described!
✅ Pocket button should go through the FxA flow (asking you to create an account if you're signed out, or recognizing you're signed into Firefox and prompting you to sign into Pocket).

✅ Main CTA button should open a modal and play the video with captions turned on, and captions should be the correct language.

Nice work on the tests too 💯

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          <p><a class="mzp-c-cta-link" href="{{ url('products.vpn.landing') }}">Erfahre mehr über Mozilla VPN</a></p>
+        </div>
+      </div>
+
+      <div class="mzp-c-picto">
+        {{ high_res_img('img/firefox/whatsnew/whatsnew97/pocket-de.jpg', {'alt': '', 'width': '460', 'height': '290', 'class': 'mzp-c-picto-image'}) }}
+        <h2 class="mzp-c-picto-heading">Tauche tief ein in faszinierende Geschichten</h2>
+        <div class="mzp-c-picto-body">
+          <p>Pocket ist deine persönliche Leseecke, in der du dir die Zeit nehmen kannst, das große Ganze zu betrachten. Zu ernsten oder lustigen, aber immer interessanten Themen.</p>
+          <p><a class="mzp-c-cta-link" href="https://getpocket.com/signup/">Erfahre mehr über Pocket</a></p>
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html:58]: (same question for FR template)

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-lg mzp-l-columns mzp-t-columns-two">
+      <div class="mzp-c-picto">
+        {{ high_res_img('img/firefox/whatsnew/whatsnew97/mozillavpn.png', {'alt': '', 'width': '460', 'height': '290', 'class': 'mzp-c-picto-image'}) }}
+        <h2 class="mzp-c-picto-heading">Privatsphäre über Firefox hinaus</h2>
+        <div class="mzp-c-picto-body">
+          <p>Schütze deine Privatsphäre auch auf Geräte-Ebene: Mit Mozilla VPN kannst du deine Verbindung zum Internet schützen und das Web dank 400 Servern in über 30 Ländern auch jenseits deines Standortes entdecken.</p>
+          <p><a class="mzp-c-cta-link" href="{{ url('products.vpn.landing') }}">Erfahre mehr über Mozilla VPN</a></p>
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html:49]: (same question for FR template)

**Comment by craigcook** [comment]
Thanks for reviewing! Updated with more analytics bits.

**Review by maureenlholland — APPROVED** [review]
LGTM r+ 🎩

**Pull request merged**